cql2pgjson/src/test/java/org/folio/cql2pgjson/ForeignKeyGenerationIT.java

```diff
@@ -154,4 +154,8 @@ public void sqlInjection2() {
     //if the test passes it means it was not successful
     assertThat(cqla("tableb.prefix == \"x0')  or 1=1)--\""), is(empty()) );
   }
+
+   public void fieldNameContainsDot() throws Exception {
+     assertThat(cqlb("copyrightStatus.name==cc"), is(empty()) );
```

Comment:
The test data should be adjusted so that the condition `copyrightStatus.name==cc` selects one out of (at least) two records.
Testing the returned record using `containsInAnyOrder(...)` proves that the code actually works.

Reply:
Simply extend https://github.com/folio-org/raml-module-builder/blob/master/cql2pgjson/src/test/resources/foreignKey.sql as needed.